### ISSUE TITLE
How to get the whiteOwnerMap of the current board state?

### ISSUE THREAD

**Issue by starshine360** [lightvector/KataGo#862]
When I use the "analyse" sub-command to analyse the current board state, I find that the whiteOwnerMap looks like the result that AI has make a next move. However, what I want is the whiteOwnerMap of the current board state. So Is there any methods to get that, please?

**Comment by starshine360** [comment]
@lightvector Thanks for your reply. Well, sometimes the players (for example, many children) are not not skilled enough. When they choose to end the game, there may be still some unsettled stones in the board. In that case, if I can get the ownership of the current board state, then I can use it to compute the size of the territory of both white and black. If black_size > white_size + komi, then the winner is black，otherwise the winner should be white. So the key is that how to get the ownership of the current board state. However, the ownership produced by katago's analyse sub-command looks like the result that AI has make a next move, as below. Since the game is end, nobody should have a next move.
![POPO-20231208-115858](https://github.com/lightvector/KataGo/assets/45736998/f605ce6c-f3cf-4c8e-a1e9-ea6955f97e4b)

**Comment by lightvector** [comment]
Cool, so it sounds like you want a "smart" method. So then, please take a look at the link I gave above. In case you missed it: https://forums.online-go.com/t/testing-autoscore-algorithms/47511/51 

You will have to write a bit of code that queries KataGo more than once, with each side to move and does some basic floodfilling based on dame and other simple rules in order to implement the experimental "v3" algorithm that is described, that someone came up with, or any other experimental algorithms that you like. These are all experimental methods, there is no single "correct" way to do this, but you can try things.

**Comment by lightvector** [comment]
Obviously, if you have questions, feel free to post in that OGS forum thread. I'm not the one that has been doing this research and experimentation on how to heuristically score "incomplete" games in a way that is still sensible, but I'm sure the people who are would be happy to answer questions to someone who is interested.
